### ISSUE TITLE
Disabling console messages?

### ISSUE THREAD

**Issue by aretasg** [nalepae/pandarallel#25]
Is there a way to disable console messages?

```
WARNING: Logging before InitGoogleLogging() is written to STDERR
I0529 15:34:22.434839  4958 store.cc:1100] Allowing the Plasma store to use up to 2GB of memory.
I0529 15:34:22.434947  4958 store.cc:1127] Starting object store with directory /dev/shm and huge page support disabled
```

**Comment by nalepae** [comment]
Unfortunately, these logs does not come from pandarallel, but from PyArrow Plasma (the lib pandarallel uses to dispatch data to different sub processes).

It seems I cannot control these logs directly...

**Comment by nalepae** [comment]
Implemented in 1.2.0.

You can use:
`pandarallel.initialize(verbose=0)`
